Mathlib/Algebra/CharP/Basic.lean

```diff
@@ -21,9 +21,10 @@ variable {R : Type*}
 
 namespace Commute
 
-variable [Semiring R] {p : ℕ} {x y : R}
+variable [Semiring R] {p : ℕ} (hp : p.Prime) {x y : R}
+include hp
```

Comment:
I think even `section`ed or `namespace`d `include`s are advised against, in favor of `include ... in`, but it's possible I'm wrong here. Admittedly the sections are short, so maybe it's fine.

Reply:
I checked in the reviewer stream on Zulip, and indeed I think we want to only have `include ... in`, so please either change it to that, or else back to explicitly writing the hypothesis in each statement.

Reply:
I don't think there is such a consensus, though?

Reply:
The reason using `include` like this is advised against is that it's really easy to accidentally include a hypothesis in a theorem that doesn't need it.

Reply:
Yes but all theorems here are about that hypothesis! And there is a linter against unused hypotheses anyway